modules/ossm-install-prerequisites.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/preparing-ossm-install.adoc
+
+[id="ossm-install-prerequisites_{context}"]
+= Meeting the {ProductName} prerequisites
+
+Before you can install {ProductName}, you must meet the following prerequisites:
+
+.Prerequisites
+
+* Possess an active {product-title} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
+* Install {product-title} version 3.11, or higher. For more information about the system and environment requirements, see the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/installing_clusters/install-config-install-prerequisites#system-requirements[OpenShift Container Platform documentation].
```

Comment:
Can you use 3.11 instead of 4.x? If you need to use 4.x, I'd move the prereqs to the assembly level so that you can include xrefs.

Reply:
Moved the prerequisites to the assembly. 
Working on removing all 3.11 content.